src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,21 @@
+const jwtService = require('../services/jwt.service');
+const { ApiError } = require('../utils/api.error');
+
+function authMiddleware(req, res, next) {
+  const authHeader = req.headers['authorization'] || '';
+  const [, accessToken] = authHeader.split(' ');
+
+  if (!authHeader || !accessToken) {
+    throw ApiError.badRequest('Token is required');
```

Comment:
Using `ApiError.badRequest` for a missing token might not be the most appropriate choice. Consider using `ApiError.unauthorized` to better reflect the nature of the error when a token is required but not provided.